### PR TITLE
Hardening

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/include/sound/routines.rb [hunk 1]
```diff
@@ -992,7 +995,7 @@ def InstallFonts(symlink, dontask)
           res = Convert.to_map(
             SCR.Execute(
               path(".target.bash_output"),
-              Ops.add(Ops.add(Directory.ybindir, "/copyfonts "), mpoint),
+              "#{File.join(Directory.ybindir, "/copyfonts")} #{mpoint.shellescape}",
```
Comment (lslezak) [inline src/include/sound/routines.rb]: NP: I'm thinking whether we should extend all `Directory` methods to accept a parameter, like `Directory.ybindir("copyfonts")` which would build the whole path. To make it more DRY...

File: src/include/sound/volume_routines.rb [hunk 2]
```diff
@@ -43,8 +45,8 @@ def setVolume(group, cardid, value)
         if Arch.sparc
```
Comment (lslezak) [inline src/include/sound/volume_routines.rb]: Ouch, SPARC is dead. We should rather remove that completely...

File: src/include/sound/volume_routines.rb [hunk 3]
```diff
@@ -224,6 +226,8 @@ def GlobExists(glob)
       out = Convert.to_map(
         SCR.Execute(
           path(".target.bash_output"),
+          # cannot escape it here as escaped glob return without asterisk expansion,
+          # but we provide input
           Builtins.sformat("echo -n %1", glob)
```
Comment (lslezak) [inline src/include/sound/volume_routines.rb]: Could we use [Dir.glob](https://ruby-doc.org/core-2.5.1/Dir.html#method-c-glob) here?

File: src/include/sound/routines.rb (hunk 1 above)
Comment (jreidinger) [reply src/include/sound/routines.rb]: maybe with more friendly name like we already have for path. Also problem with that ybindir is that it does not respect Y2DIR. So I prefer stuff like find_bin("copyfonts") which will return path or raise error.

File: src/include/sound/volume_routines.rb (hunk 2 above)
Comment (jreidinger) [reply src/include/sound/volume_routines.rb]: I think half of sound is dead like that fancy script to copy sound blaster.

File: src/include/sound/volume_routines.rb (hunk 3 above)
Comment (jreidinger) [reply src/include/sound/volume_routines.rb]: Well, it would be the best solution, but much bigger impact, so higher risk of bugs.

**Pull request merged**